### DISCUSSION TITLE
remnote官方是不是有一些可以下载的卡组?第一次打开的时候添加了一个, 后边就找不到了

### DISCUSSION THREAD

**Discussion by lalulalulearing** [5eagull/RemNote-Chinese-chat-group#53] [你问我答/互相帮助]
![image](https://github.com/user-attachments/assets/5b852cd6-447f-447e-ba0e-99d202d2af2d)
就是这个位置的卡组, 在哪里可以重新找到呢

**Comment by staygoldcowboy** [comment]
点头像 然后点出现的shared flashcards 之后搜索或者按主题找你要的卡组就ok了 有直接下载的按钮
![image](https://github.com/user-attachments/assets/a894de69-413b-4516-920d-51ec220f6b5a)
